django/db/models/base.py

```diff
@@ -204,7 +204,7 @@ def __new__(cls, name, bases, attrs):
             if base not in parents or not hasattr(base, '_meta'):
                 # Things without _meta aren't functional models, so they're
                 # uninteresting parents.
-                inherited_attributes |= set(base.__dict__.keys())
+                inherited_attributes.update(base.__dict__.keys())
```

Comment:
Just out of interest, `dict.keys()` is a **set-like** object providing a **view** on a dictionary's keys in Python 3. So the cast to `set` was only needed for Python 2 where `dict.keys()` returns a list.

Thus the options here are changing to use `set.update()` and passing the dictionary as-is - as has already been committed, or to just drop the cast to `set` as it already is set-like which might look a bit less magic. No strong opinion on this however.